README.md

```diff
@@ -37,7 +37,10 @@ docker-compose up
 
 ## Indexing
 
-`try-cloaked-search` includes some test data. Since Cloaked Search uses a different key per (tenant, index, field), documents with protected fields must be tagged with the tenant they belong to. Half of the articles in the test dataset are associated with `tenant-1`, and the other half are not associated with any tenant. Only documents belonging to `tenant-1` will be encrypted. To better understand Cloaked Search's key management, refer to the [configuration documentation](/docs/saas-shield/cloaked-search/configuration).
+`try-cloaked-search` includes some test data. Since Cloaked Search uses a different key per (tenant, index, field),
+documents with protected fields must be tagged with the tenant to which they belong.
+Half of the articles in the test dataset are associated with `tenant-1`, and the other half are not associated with any tenant.
+Only documents belonging to `tenant-1` will be encrypted. To better understand Cloaked Search's key management, refer to the [configuration documentation](/docs/saas-shield/cloaked-search/configuration).
```

Comment:
This link is not correct. I think you meant to link to https://ironcorelabs.com/docs/saas-shield/cloaked-search/configuration/

Reply:
Oops - I was thinking of the website pages..